tests/unit/initializers/route-alias-test.js

```diff
@@ -13,7 +13,7 @@ module('Unit | Initializer | route alias', {
       application = Ember.Application.create();
       application.deferReadiness();
       RouteAliasInitializer.initialize(application);
-      DSL = new Ember.RouterDSL(null, {});
+      DSL = new Ember.RouterDSL('application', {});
```

Comment:
We now need to set the 'parent' name to `application` here --

In this commit https://github.com/emberjs/ember.js/commit/6d9a9735eeb04182589ff9bd98569288667cb8d8#diff-57d1dbf7bb82d085435a5e830a2e77c7L177 ember.js removed the check to see if `dsl.parent` is truthy. Because of that, it is possible to generate route names that look like `null.index` instead of `index`, as expected if no parent is specified.